### ISSUE TITLE
Metadata for Quicklisp

### ISSUE THREAD

**Issue by hydandata** [mabragor/cl-yaclyaml#4]
Please consider adding :description, :author and :license information to your ASDF system(s). This will greatly help Quicklisp users and make it easier for them to report bugs.

More information:

http://blog.quicklisp.org/2015/05/looking-for-more-metadata.html
https://www.quicklisp.org

**Comment by mabragor** [comment]
It appears that cl-yaclyaml.asd has licence, author and description fields.
Should I add some more descriptive description or something?

**Comment by hydandata** [comment]
Yes please, so that when user does apropos they can see it. Thanks!
